src/SongInfo.cpp

```diff
@@ -0,0 +1,44 @@
+//
+// Created by shvmo on 03/05/2020.
+//
+
+#include "SongInfo.h"
+
+namespace DataStructuresWet1 {
+    SongInfo::SongInfo() = default;
```

Comment:
can move to header file

Reply:
according to this it's better to implement in the body. As a good practice: https://stackoverflow.com/questions/43733938/default-c-definitions-in-body-or-header-files